hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeVertex.java

```diff
@@ -460,10 +462,14 @@ public <V> VertexProperty<V> property(
             E.checkArgument(!this.hasProperty(propertyKey.id()),
                             "Can't update primary key: '%s'", key);
         }
+        if (value == null) {
+            this.removeProperty(propertyKey.id());
+            return EmptyVertexProperty.instance();
+        }
```

Comment:
@javeme already exist?

Reply:
ok, maybe I didn't notice the new commit